### PR TITLE
Add State of the Moustache blog post

### PR DESCRIPTION
**Pull request by addyosmani**
review: @yeoman/owners

### CODE REVIEW

**Comment by hemanth** [comment]
:+1: Awesomeness++ :)

**Comment by arthurvr** [comment]
Nice work! :trophy:

**Comment by ragingwind** [comment]
Wow! amazing.

**Comment by SBoudrias** [comment]
Very nice!

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: State of the Moustache
+excerpt: Digest of what's new in the Yeoman ecosystem
+---
+
+> this is a post by team member [Addy Osmani](https://github.com/addyosmani). <br>In this issue we cover generators for React, ES6, desktop apps and more.
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: This

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+### Spotlight: ES6
+
+A number of new generators are also now available for quickly turning your ES6+ code into ES5 code. There are generators available for [Babel](https://github.com/thejameskyle/generator-es6-library-boilerplate), [Angular2](https://github.com/swirlycheetah/generator-angular2) and [Aurelia](https://github.com/zewa666/generator-aurelia) - a new framework using ES6 and Web Components. A few more ES6 generators can be found over on my [es6-tools](https://github.com/addyosmani/es6-tools#code-generation) project.
+
+![](/assets/img/blog/april-es62.png)
+
+### Spotlight: Electron
+
+[Electron](http://electron.atom.io/), formerly atom-shell, allows you to build cross-platform desktop apps using JS/CSS/HTML. Some of the apps built using it that you may already be using include [Slack](https://slack.com/), [Atom Editor](https://atom.io) (of course) and PhotoShop/Sketch export tool [Avocode](http://avocode.com/).
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: PhotoShop => Photoshop

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+### Articles
+
+[Multi-User Video Conference with WebRTC and Yeoman](http://blog.mgechev.com/2014/12/26/multi-user-video-conference-webrtc-angularjs-yeoman)
+
+[Writing Yeoman generators in ES6](http://mammal.io/articles/yeoman-generators-es6)
+
+[Putting your bot on #Slack using Yeoman](http://blog.pandorabots.com/putting-your-bot-on-slack/)
+
+[How to Build an App With AngularJS, Node.js and Stormpath in 15 Minutes](https://stormpath.com/blog/angular-node-15-
+minutes)
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: Shouldn't be a hard-wrap here.

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+[Worflow automation for Wordpress Theme Developers with Yeoman](https://speakerdeck.com/artificermil/front-end-workflow-automation-for-wordpress-theme-and-plugin-development)
+
+[A Baseline for Front-End [JS] Developers: 2015](http://rmurphey.com/blog/2015/03/23/a-baseline-for-front-end-developers-2015)
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: `[JS]` => `\[JS\]`

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+[Yeoman, Polymer and Gulp](http://robdodson.me/yeoman-polymer-and-gulp)
+
+[Integrating Yeoman with third-party tools](http://yeoman.io/authoring/integrating-yeoman.html)
+
+[Using yo webapp via Docker](https://asciinema.org/a/13240)
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: ```Using `yo webapp` via Docker```

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+[How to Wire Up Ruby on Rails and AngularJS as a Single-Page Application (Gulp Version)](http://angularonrails.com/how-to-wire-up-ruby-on-rails-and-angularjs-as-a-single-page-application-gulp-version/?utm_medium=email&utm_source=rubyweekly)
+
+[Practical TypeScript Development with Gulp and Sublime Text 3](https://airpair.com/typescript/posts/typescript-development-with-gulp-and-sublime-text?utm_medium=email&utm_source=javascriptweekly)
+
+### Some generators we liked
+
+[A new generator for Django projects](http://axiacore.github.io/generator-django-axiacore/)
+
+[Mobile Chrome Apps Starter Kit with Polymer](http://goo.gl/3MdaIk)
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: Use the full URL

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+### Composition FTW
+
+It's been pleasant to see an [increasing](https://twitter.com/tomwayson/status/588564830665084930) number of developers using our [composability](http://yeoman.io/authoring/composability.html) feature for their generators. This enables building upon common ground so that a generator can be run inside another generator, rather than having to replicate functionality. E.g call out to a generator for scaffolding Karma tests rather than doing this inside your AngularApp one.
+
+## Tipping our hat
+
+That's about it for this round-up. We would like to remind folks that we're always checking out [@yeoman](http://twitter.com/yeoman) and enjoy hearing suggestions for anything we can be doing better, news, new generators you've written or even just [doodles](https://twitter.com/kittytail38/status/573522972528373761) of your favorite man in a hat. Until next time, `yo awesomeness`.
+
```
Comment (sindresorhus) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: Maybe encourage people to comment interesting stuff for the next post here: https://github.com/yeoman/yeoman/issues/1411

**Comment by addyosmani** [comment]
@sindresorhus Thanks for the feedback! Pushed a commit with fixes.

File: app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+### Composition FTW
+
+It's been pleasant to see an [increasing](https://twitter.com/tomwayson/status/588564830665084930) number of developers using our [composability](http://yeoman.io/authoring/composability.html) feature for their generators. This enables building upon common ground so that a generator can be run inside another generator, rather than having to replicate functionality. E.g call out to a generator for scaffolding Karma tests rather than doing this inside your AngularApp one.
+
+## Tipping our hat
+
+That's about it for this round-up. We would like to remind folks that we're always checking out [@yeoman](http://twitter.com/yeoman) and enjoy hearing suggestions for anything we can be doing better, news, new generators you've written or even just [doodles](https://twitter.com/kittytail38/status/573522972528373761) of your favorite man in a hat. If you have ideas for the next post, you can also drop em' in this [thread](https://github.com/yeoman/yeoman/issues/1411).
```
Comment (passy) [inline app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md]: drop 'em :)

**Comment by passy** [comment]
Excellent work, Addy! :ship: it!

**Comment by silvenon** [comment]
Nice, Addy!

**Comment by addyosmani** [comment]
Thanks for reviewing, all! :) @passy I've fixed the nit you mentioned too. Landing this. Wooooooo.

**Pull request merged**